rollup.config.mjs

```diff
@@ -91,20 +106,25 @@ export default {
           "write-file-atomic",
           /.*\/getFileIgnorer/,
           /.*\/FileCache/,
-        ].map(find => ({find, replacement: require.resolve("./shim/empty")}))
+          /.*\/resolveSilent/,
+        ].map(find => makeShim(find, "./shim/empty"))
       ]
     }),
     resolve(),
     json(),
     cjs({nested: true}),
     inject({
-      process: require.resolve("./shim/process")
+      process: resolvePkg("./shim/process")
     }),
     iife({
       names: id => {
         if (/stylelint-bundle\.min\.js/.test(id)) {
           return "stylelint";
         }
+        if (/^node:/.test(id)) {
+          // we only modify internal id so we can get an error when an external id goes wrong
+          id = id.replace(/^node:/, 'node_');
+        }
         return `_external_${id}`;
```

Comment:
The error was generated by this line when `id` is not a simple word.

After this PR:
https://github.com/stylelint/stylelint/pull/7279

Stylelint uses `import()` to load modules, making rollup generate multiple chunks into `dist`, and the id of generated chunks is their full file path.

This was fixed by setting `inlineDynamicImports: true`.